src/build_defs.bzl

```diff
@@ -382,6 +306,19 @@ def mozc_win32_cc_prod_binary(
       visibility: optional. The visibility of the target.
       **kwargs: other arguments passed to mozc_cc_binary.
     """
+    mandatory_target_compatible_with = [
```

Comment:
Please move this block (line 309 - 321) after mozc_cc_binary (line 322 - 336) to keep the consistency with the current implementation. It causes an error in my environment.

Reply:
Done. Thank you for the investigation.